app/views/home/index.slim

```diff
@@ -32,4 +36,36 @@ section.card.card-2.mx1.mb1
           div.a.card-divider.block.text-decoration-none.py2 href="#{post_path(post)}"
             div.table-cell.featured-posts.px1
               h5.h5.m0.card-listItem-header=post.title
-              p.m0.card-listItem-text=TruncateContent(post.content)
+              p.m0.card-listItem-text=truncate_content(post.content)
+
+//Footer links
+section.footer.mx1.mb1
+  ul.list-reset.h6
+    li
```

Comment:
Should we create a helper method with an array of an object with attributes title, description, url. And then loop over the array here?